cli/config/config.go

```diff
@@ -173,6 +173,17 @@ func initConfig() {
 	} else {
 		Source = "default"
 	}
+	if len(Path) > 0 {
```

Comment:
nit: do we need to check `len(Path) > 0`? wouldn't the next statement suffice

Reply:
oh, good point, we can remove that